charts/spire/charts/spire-server/values.yaml

```diff
@@ -628,6 +628,12 @@ customPlugins:
   upstreamAuthority: {}
   notifier: {}
 
+experimental:
+  ## @param experimental.cacheReloadInterval The amount of time between two reloads of the in-memory entry cache.
+  cacheReloadInterval: 5s
```

Comment:
This causes it to always be set in the config file. Seems like it should't be set in the config file until explicitly set by the user since experimental features most likely shoulnd't be enabled in prod?

Reply:
Good point. What do you think of an `enabled` flag for this section?

Reply:
Yeah, I think that would probably work. Makes it explicit that the user is intentionally using it.

Reply:
enabled flag added